app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -54,6 +54,13 @@ def stored_result
       @stored_result = document_capture_session&.load_result
     end
 
+    def liveness_checking_enabled?
+      return false if Identity::Hostdata.env == 'prod'
+      return false unless IdentityConfig.store.doc_auth_selfie_capture_enabled
+
+      decorated_sp_session.selfie_required?
```

Comment:
I feel like the double-checking of both hostdata and config implies a distrust in our configuration, where we should be confident in using it? Assuming the default is `false` in production.

```suggestion
      IdentityConfig.store.doc_auth_selfie_capture_enabled && decorated_sp_session.selfie_required?
```

Reply:
@aduth - I understand that the FF is there and reliable.  However, given the sensitivity of selfie not yet ready for our higher envs, we want it to be impossible for this to go live in production as this is an explicit requirement.  No matter what config changes are made, this hardcoding guards any mistakes from a config standpoint.

Reply:
`decorated_sp_session.selfie_required?` will also return false if `IdentityConfig.store.doc_auth_selfie_capture_enabled` is not set to true. I understand about wanting to make triple-sure this doesn't go live accidentally, and at the same time code encapsulation is also useful. Maybe we could put any extra checks in selfie_required and just use that everywhere instead of having an extra method `liveness_checking_enabled?`. Happy to rename the method if that would help.

Reply:
I agree with Amir.  Since this is such a sensitive feature, I think it's good to have an extra safeguard in place.